tests/unit/test_formatter.py

```diff
@@ -272,3 +274,29 @@ def test_format_exception():
     assert result.endswith(
         'test_format_exception\n    raise Exception("error")\nException: error'
     )
+
+
+def test_override_format_exception():
+    class CustomFormatter(Formatter):
+        def formatException(self, ei) -> str:
```

Comment:
Still the format should be overriden I think because of:
https://github.com/microsoft/picologging/blob/783acf1eba13fd587028a8dcd0d97544d33149a2/src/picologging/handler.cxx#L99-L103